### PR TITLE
[TMVA][SOFIE] Support for parsing Softmax, LeakyRelu, and Tanh in Keras parser

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds support for parsing activation functions like Softmax, LeakyRelu and Tanh in SOFIE Keras parser.

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default with flags -Dtmva-sofie=On

**Comment by sanjibansg** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On
